theurgy.lic

```diff
@@ -62,6 +62,69 @@ def clean_altar
   put_in_container($settings.water_holder)
 end
 
+def prayer_bead
```

Comment:
In that case you'll want to add a check to the `prayer_bead` method against `DRStats.circle` (see `comm_eluned` for an example.